src/components/SidePanels/OutlinerPanel/OutlinerPanel.tsx

```diff
@@ -26,6 +26,10 @@ const OutlinerPanelComponent: FC<OutlinerPanelProps> = ({ regions, ...props }) =
     regions.setFilteredRegions(value);
   }, [regions]);
 
+  const hiddenRegions = useMemo(() => {
+    return regions?.regions?.length - regions?.filter?.length;
```

Comment:
```suggestion
    return (regions?.regions?.length ?? 0) - (regions?.filter?.length ?? 0);
```

Reply:
I can't do this because if regions.regions.length is 2 and regions.filter.length is undefined the hiddenRegions would return 2 even with no region filtered. But I change the way that hiddenRegions is returning.